src/components/MessagesList/MessagesList.vue

```diff
@@ -429,7 +431,8 @@ export default {
 
 			// Make the request
 			try {
-				const messages = await request({ token, lastKnownMessageId, includeLastKnown: includeLastKnown ? '1' : '0' })
+				this.oldMessagesPromise = request({ token, lastKnownMessageId, includeLastKnown: includeLastKnown ? '1' : '0' })
```

Comment:
the component is reused, so you need to store this based on token?
Otherwise if you quickly change to 2 unloaded conversations the second one wouldn't load?

Reply:
Or more simply reset the value to null when the `token` changes?

Reply:
if you switch to another conversation the call will get cancelled anyway, so you'll then get a new request